File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/")

    title=driver.title
    print("Title of the page is: ",title)

    driver.find_element(By.LINK_TEXT, "About Us").click()
    newTitle=driver.title
    print("Title of the page is: ",newTitle)

File: Selenium/Project/Activity1.java
package alchemyJobs;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity1 {
	
	WebDriver driver;
	
	@BeforeClass
	public void setUp() {
	    driver=new FirefoxDriver();
	    driver.get("https://alchemy.hguy.co/jobs/");
	}
	
	@Test
	public void verifyTitle() {
		String title=driver.getTitle();
		Assert.assertEquals(title, "Alchemy Jobs – Job Board Application","Page title is incorrect");	
	}
	
	@AfterClass
	public void tearDown() {
		driver.quit();
	}
	

}
